### ISSUE TITLE
Dead code cleanup

### ISSUE THREAD

**Issue by micrococo** [lxqt/qps#242]
**qps** has a lot of dead code:

- Code in conditional blocks with conditions that are always `false`.
- Unused variables, functions and classes.
- Commented code and functions with all of its code commented.

Are pull request removing that code accepted or is it kept for something? And what about MOSIX (that is a proprietary operating system) specific code?

**Comment by tsujan** [comment]
We're aware of that and have removed some redundant codes.

So far, our focus was on making Qps really work fine and that's done. Now, it's time for serious code cleanup, which will take a lot of time -- and is boring ;) It should be done with care; we don't want to damage the functionality. In some places, the whole code structure may need to be changed.

**Comment by micrococo** [comment]
Yes, removal should be made with care and in several stages. I'm not profient in C++ but recently removed all the unreachable code without any noticeable issues. That made me think that it could be removed safely, but again I'm not profient in C++. I made my own task manager some time ago as an exercise to learn a bit of C++ and Qt. It works but it is dead simple compared to qps.

![captura_20200422_215647](https://user-images.githubusercontent.com/7717273/80027555-3aac4600-84e4-11ea-96d5-2c2646aee953.png)

I've tried to understand the code of qps but I am not able to do so. To the eye of a noob like me, it looks like a rewrite from scratch could take less time than fixing the current code. :)

**Comment by tsujan** [comment]
>  it looks like a rewrite from scratch could take less time than fixing the current code. :)

I don't know about other devs but, when I was fixing Qps, I had similar thoughts ;)

Anyway, if you make cleanup PRs -- which would be VERY appreciated -- please make them so that they can be reviewed as easily as possible.

**Comment by micrococo** [comment]
Understood. Thanks.

**Comment by shlyakpavel** [comment]
>  I made my own task manager some time ago as an exercise to learn a bit of C++ and Qt. It works but it is dead simple compared to qps.

Why not just to upload it on GitHub? Maybe there will be some people involved in development.
